### DISCUSSION TITLE
Spoon

### DISCUSSION THREAD

**Discussion by bhargavkumar7031** [hyf-Group2-fp/Just4Giving#61] [Show and tell]
Hi All, I had a similar idea where food is donated to people who are in need of. In India there are NGO's working as supply chain to donate food and there are people who are willing to donate but they don't know whom to contact. platform would bridge between donor and NGO's. I believe im good working with javascript and its frameworks so is there any opportunity for me?

[spoon](https://spoonate.netlify.app/)

inspired by [food waste](https://www.youtube.com/watch?v=6RlxySFrkIM)

**Comment by bermarte** [comment]
> platform would bridge between donor and NGO's
Thanks @bhargavkumar7031 for the input, I will talk about that with the others.
